src/SFA.DAS.Reservations.Api.Client/HttpHelper.cs

```diff
@@ -49,5 +41,26 @@ private async Task<T> GetAsync<T>(string url, object data)
                 throw;
             }
         }
+
+        public async Task<TResponse> PostAsJson<TRequest, TResponse>(string url, TRequest data, CancellationToken token)
+        {
+            token.ThrowIfCancellationRequested();
+
+            string stringResponse = null;
+            try
+            {
+                stringResponse = await PostAsync(url, JsonConvert.SerializeObject(data));
```

Comment:
It would be great if the SFA.DAS.Http client accepted the cancellation token, though whether it's worth adding an overload to the client is debatable. Especially since this is an old version of the nuget, which would be a pain to update.

Reply:
Agreed